### PR TITLE
show only logs with a severity level of ERROR or higher in the stderr

### LINKED ISSUE #4049: metrics-apiserver json logs
> I'm very sorry @JorTurFer - I was looking at logs from the `keda-operator-metrics-apiserver` deployment. It seems that only has a level setting as opposed to the operator having level, format, and timeEncoding.
> 
> Our logs for `keda-operator` are correctly being emitted as JSON 🤦

That said, I'd like to have the metrics-apiserver log in json format as well, primarily because all info logs are emitted to stderr and those show up as error logs in our logging system. Could this functionality please be added?

_Originally posted by @naseemkullah in https://github.com/kedacore/keda/issues/3655#issuecomment-1367852952_

### PR DESCRIPTION
**Pull request by Adarsh-verma-14**
Signed-off-by: Adarsh-verma-14 [t_adarsh.verma@india.nec.com](mailto:t_adarsh.verma@india.nec.com)

In the logs of keda-metrics-apiserver, show only printing errors, but errors should go to stderr.

### Checklist

- [x] Commits are signed with Developer Certificate of Origin (DCO - [learn more](https://github.com/kedacore/keda/blob/main/CONTRIBUTING.md#developer-certificate-of-origin-signing-your-work))

Fixes #4049

### CODE REVIEW

**Comment by Adarsh-verma-14** [comment]
#4882 this PR is closed due error from my side and again raised new PR regarding same things

File: CHANGELOG.md [hunk 1]
```diff
@@ -91,6 +91,7 @@ New deprecation(s):
 - **GitHub Runner Scaler**: Fix explicit repo check 404 to skip not crash ([#4790](https://github.com/kedacore/keda/issues/4790))
 - **GitHub Runner Scaler**: Fix rate checking on GHEC when HTTP 200 ([#4786](https://github.com/kedacore/keda/issues/4786))
 - **Pulsar Scaler**: Fix `msgBacklogThreshold` field being named wrongly as `msgBacklog` ([#4681](https://github.com/kedacore/keda/issues/4681))
+- **General**: Add one deployment arg in keda-metrics-apiserver for only printing errors, but errors should go to stderr ([#4049](https://github.com/kedacore/keda/issues/4049))
```
Comment (zroubalik) [inline CHANGELOG.md:94]: Could you please move this up to Unreleased section, to ### Other subsection? Thanks!

Comment (Adarsh-verma-14) [reply CHANGELOG.md]: sure @zroubalik , actually there are some problem with my side and branches during update, so I am closing this PR and will again raised regarding same things

Comment (Adarsh-verma-14) [reply CHANGELOG.md]: I am sorry for  this inconvenience .

**Pull request closed without merging**